app/src/main/java/com/keylesspalace/tusky/view/ExposedPlayPauseVideoView.kt

```diff
@@ -0,0 +1,33 @@
+package com.keylesspalace.tusky.view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.VideoView
+
+class ExposedPlayPauseVideoView : VideoView {
+    private var mListener: PlayPauseListener? = null
+
+    constructor(ctx: Context) : super(ctx)
+    constructor(ctx: Context, attrs: AttributeSet) : super(ctx, attrs)
+    constructor(ctx: Context, attrs: AttributeSet, defStyleAttr: Int) : super(ctx, attrs, defStyleAttr)
+    constructor(ctx: Context, attrs: AttributeSet, defStyleAttr: Int, defStyleRes: Int) : super(ctx, attrs, defStyleAttr, defStyleRes)
```

Comment:
use one overloaded constructor for this (like here https://github.com/tuskyapp/Tusky/blob/develop/app/src/main/java/com/keylesspalace/tusky/view/PollPreviewView.kt#L26-L30 )